test/runtests.jl

```diff
@@ -64,4 +64,8 @@ end
     if !is_APPVEYOR && GROUP == "GPU"
         @safetestset "NNPDE_gpu_Lux" begin include("NNPDE_tests_gpu_Lux.jl") end
     end
+
+    if GROUP == "All" || GROUP == "Other algos"
```

Comment:
```suggestion
    if GROUP == "All" || GROUP == "DGM"
```

Reply:
The reason I had it this way was that other algs like Deep Ritz can also be added to the same group.